src/formatting/visitor.rs

```diff
@@ -486,7 +486,8 @@ impl<'b, 'a: 'b> FmtVisitor<'a> {
             // For use/extern crate items, skip rewriting attributes but check for a skip attribute.
             ast::ItemKind::Use(..) | ast::ItemKind::ExternCrate(_) => {
                 if contains_skip(attrs) {
-                    self.push_skipped_with_span(attrs.as_slice(), item.span(), item.span());
+                    // `item.span()` includes the attributes, while `item.span` does not.
```

Comment:
These comments are not necessary, so lets remove them. The span field on the AST node by design doesn't include attributes. The `span()` function comes from one of our traits which provides a span that includes the attributes, where applicable

https://github.com/rust-lang/rustfmt/blob/87e960271b4036d37f73490bec8cba67f93b94ed/src/formatting/spanned.rs#L12-L15

```suggestion
```

Reply:
Done